templates/jquery/js/tree-grid/tree-grid-editing/files/index.html

```diff
@@ -26,26 +26,7 @@
 			   ],
 			   childDataKey: "products",
 			   features: [
-				   {
-						name: "Updating",
-						columnSettings: [
-							{
-								columnKey: "progress",
-								editorType: "currency",
-								editorOptions: {
-									buttonType: "spin"
-								}
-							},
-							{
-								columnKey: "start",
-								editorType: "datepicker"
-							},
-							{
-								columnKey: "finish",
-								editorType: "datepicker"
-							}
-						]
-				   }$(treeGridFeatures)
+				  $(treeGridFeatures)
```

Comment:
Leave that one as it is, until we refactor the tree grid helper too so those default Updating settings can be moved over.